README.md

```diff
@@ -2,4 +2,18 @@
 
 [![Join the chat at https://gitter.im/jupyterhub/binder](https://badges.gitter.im/jupyterhub/binder.svg)](https://gitter.im/jupyterhub/binder?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Binder metapackage for usage, docs, and chat
+
+**For the main binder repository, visit [binderhub](https://github.com/jupyterhub/binderhub)**
+
+This repository is the metapackage for [binder](https://mybinder.org) usage, docs, and chat.
+
+ Major binder development happens on [binderhub](https://github.com/jupyterhub/binderhub).  The majority of issues related to binder at this repo.  Please refer to BinderHub for further details on the binder project.
+
+## Additional repos
+
+ For deployment of the website [mybinder.org](https://mybinder.org), visit [mybinder.org-deploy](https://github.com/jupyterhub/mybinder.org-deploy).  Helm charts are at [helm-chart](https://github.com/jupyterhub/helm-chart).  
```

Comment:
`BinderHub` uses technology called "Helm" to configure and manage deployment of the Binder service. For details about this deployment, see [helm-chart]...` etc